src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MethodTypeInference.cs

```diff
@@ -2364,6 +2458,61 @@ private void UpperBoundTypeArgumentInference(NamedTypeSymbol source, NamedTypeSy
             targetTypeArguments.Free();
         }
 
+#nullable enable
+        private bool UpperBoundFunctionPointerTypeInference(TypeSymbol source, TypeSymbol target, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
+        {
+            if (source is not FunctionPointerTypeSymbol { Signature: { } sourceSignature } || target is not FunctionPointerTypeSymbol { Signature: { } targetSignature })
```

Comment:
>if (source is not FunctionPointerTypeSymbol { Signature: { } sourceSignature } || target is not FunctionPointerTypeSymbol { Signature: { } targetSignature }) [](start = 12, length = 157)

I am not sure where we deal with the conversion part from the specification: " `V` is a function pointer type which has an implicit function pointer conversion to `delegate*<U2..Uk, U1>`."

Reply:
The specification no longer has that section.

---
In reply to: [558415248](https://github.com/dotnet/roslyn/pull/50249#discussion_r558415248) [](ancestors = 558415248)